### PR TITLE
added apple web app capable coding

### PR DESCRIPTION
**Pull request by RageInvader**
first part allows it to become a web app when adding to home screen

the script part is to stop href links breaking out to safari, as web apps need to be javascript links

### CODE REVIEW

File: client/head.html [hunk 1]
```diff
@@ -7,4 +7,14 @@
 	<!-- <link rel="stylesheet" href="/css/bootstrap.min.css"> -->
     <!-- <link rel="stylesheet" href="/css/styles.css"> -->
     <link rel="icon" href="favicon.ico">
+    
+    <!-- Apple Web App Capable -->
+    <meta name="apple-mobile-web-app-capable" content="yes">
```
Comment (lokenx) [inline client/head.html]: How is this different from whats on line 3?

**Comment by lokenx** [comment]
What's the un-minified version of the script take you've got there?

File: client/head.html (hunk 1 above)
Comment (RageInvader) [inline client/head.html]: for iOS devices it needs to be;  meta name="apple-mobile-web-app-capable"

**Comment by RageInvader** [comment]
I'm unsure, I used the code from here (https://github.com/h5bp/mobile-boilerplate/blob/master/index.html) as after making my web app, all my links were plain html 'href' format, and kept loading in safari and breaking out the app. If you already use javascript for your url's you likely won't need it.

**Pull request closed without merging**